template-for-repository/proofs/arpa-test.sh

```diff
@@ -0,0 +1,342 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script performs  data collection for Arpa.
+# Furthermore, it serves as a validation framework for existing CBMC proofs.
+
+# TODO add project-specific list of proofs that fail, but should be overriden due to limitations of the validation approach
```

Comment:
Talk to Kareem, but I'm not sure TODO lists belong in public repositories.  But I may violate this rule myself...